packages/console/src/pages/SignInExperience/hooks/use-custom-phrases-context.ts

```diff
@@ -33,13 +36,16 @@ export type Context = {
 };
 
 export const CustomPhrasesContext = createContext<Context>({
+  defaultLanguageTag: 'en',
+  existedLanguageTags: [],
```

Comment:
it seems that `languageTags` is clear enough

Reply:
Just for clarification. We have many `language` related things in the context, E.g., `builtInLanguages`, `customLanguages`, `preAddedLangauge`, `preSelectedLanguage`, `displayingLanguages`, and so on.

In summary, 
`existedLanguageTags` = `builtInLanguages` + `customLanguages`;
`displayingLanguages` = `existedLanguageTags` + `preAddedLangauge`;

So I give these 'languages' different names related to their context.